File: Tl2.4 RESTFULL_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_restful_dotnet/ec.edu.monster.servicio/LoginService.cs
namespace ec.edu.monster.servicio;

public class LoginService
{
    public bool Login(string? username, string? password)
    {
        if ((username == "MONSTER" && password == "MONSTER9") ||
            (username == "admin" && password == "admin"))
        {
            return true;
        }
        return false;
    }
}

File: Tl2.3 RESTFULL_JAVA_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_restful_java/src/main/java/ec/edu/monster/servicio/LoginService.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.servicio;

/**
 *
 * @author crist
 */
public class LoginService {
        public boolean login(String username, String password) {
        if ((username.equals("MONSTER") && password.equals("MONSTER9"))
                || (username.equals("admin") && password.equals("admin"))) {
            return true;
        }
        return false;
    }
}
